### PR TITLE
Line length feature flag

### PR DESCRIPTION
**Pull request by contolini**
This is an attempt to move some CSS behind a Wagtail feature flag. See https://[GHE]/CFGOV/platform/issues/237 for line length background.

## Additions

- Feature flag gulp task.
- Feature flag CSS directory to put relevant stylesheets in.
- Feature flag conditional at the bottom of `base.html`.

## Testing

- Pull down branch.
- `gulp styles`
- Log into the wagtail admin and add a `NEW_LINE_LENGTH` feature flag.

screen shot 2016-08-30 at 4 28 53 pm

When it's enabled (most) lines should wrap after `41.875rem`. For example, try visiting http://localhost:8000/office-civil-rights/eeo-policy-and-reports/ before and after toggling the `NEW_LINE_LENGTH` checkbox.

## Notes
- After the feature has been reviewed by stakeholders we'll remove this flag and merge the CSS into the appropriate stylesheet.

## Review

- @rosskarchner 
- @virginiacc 
- @sebworks 

## Checklist

* [ ] Changes are limited to a single goal (no scope creep)
* [ ] Code can be automatically merged (no conflicts)
* [ ] Code follows the standards laid out in the [front end playbook](https://github.com/cfpb/front-end)
* [ ] Passes all existing automated tests
* [ ] New functions include new tests
* [ ] New functions are documented (with a description, list of inputs, and expected output)
* [ ] Placeholder code is flagged
* [ ] Visually tested in supported browsers and devices
* [ ] Project documentation has been updated (including the "Unreleased" section of the CHANGELOG)

### CODE REVIEW

**Comment by rosskarchner** [comment]
me right now: http://www.gettyimages.com/license/465117348

**Comment by sebworks** [comment]
👍 , looks great. Should we set the `max-width` on `.blocks` as well?

File: cfgov/unprocessed/css/feature-flags/line-length.less [hunk 1]
```diff
@@ -0,0 +1,11 @@
+// Experimenting with a capped line length across all website content.
+.content_main {
+    h3,
+    h4,
+    h5,
+    h6,
+    p,
+    li {
```
Comment (Scotchester) [inline cfgov/unprocessed/css/feature-flags/line-length.less]: add `dt` and `dd` to the list

**Comment by contolini** [comment]
> 👍 , looks great. Should we set the max-width on .blocks as well?

Ehh I'll let @Scotchester or @anselmbradford answer that one.

**Comment by KimberlyMunoz** [comment]
> Should we set the max-width on .blocks as well?

My instinct is no, because we might get some unexpected behavior on blocks with backgrounds and borders if applied directly to the block. Also content main covers almost all the big blocks of text unless there's a block in the sidebar, navigation, hero homepage or footer, so it doesn't seem necessary.

**Comment by Scotchester** [comment]
I agree with @KimberlyMunoz's answer.

**Comment by sebworks** [comment]
> I agree with @KimberlyMunoz's answer.

👍

**Pull request merged**